src/StyledFormComponents.js

```diff
@@ -17,12 +18,21 @@ const StyledCardHeading = styled.h3`
 `;
 
 const StyledCard = styled.div`
+  ${props =>
+    props.favouritesCard
+      ? `
+  width: 45%;
+  height: 80vh;
+  margin: 10px;
+  display: inline-block;
+  `
+      : `width: 900px;
+  height: 450px;
+  margin: 0 auto;`}
   border-style: solid;
   border-color: black;
   border-radius: 25px;
   background: #dbd8db;
-  width: 900px;
```

Comment:
you could also write these as ` width: ${ ({ favouritesCard }) =>  favouritesCard ? '45%' : '900px' }` and same for the others!